lib/resque-serializer.rb

```diff
@@ -9,6 +11,16 @@
 module Resque
   module Plugins
     module Serializer
+      class << self
+        attr_accessor :configuration
+
+        def configure
+          self.configuration ||= Configuration.new
```

Comment:
Maybe use singleton on  `Configuration` class and use `Configuration.instance` in here.